immuno_graphical/verification/immuno_graphical_verification_code.R

```diff
@@ -0,0 +1,470 @@
+weighted_cdf("bindRBD","Baseline Neg")
+weighted_cdf("pseudoneutid50","Baseline Neg")
+weighted_cdf("pseudoneutid80","Baseline Neg")
+weighted_cdf("bindSpike","Baseline Pos")
+weighted_cdf("bindRBD","Baseline Pos")
+weighted_cdf("pseudoneutid50","Baseline Pos")
+weighted_cdf("pseudoneutid80","Baseline Pos")
+
+
+
+
+#Boxplots of D29 Ab markers: baseline negative vaccine + placebo arms
+
+box_plot <- function(assay1,Trt1){
+  boxplot_data <- dat.long.twophase.sample %>%
+  filter(assay == assay1,
+         Bserostatus == "Baseline Neg",
+         Trt == Trt1) 
+  write.csv(data.frame(summary=names(summary(boxplot_data$Day57)),value=matrix(summary(boxplot_data$Day57))),
+            here(paste("immuno_graphical/verification/output/output_tester/boxplots_day57_summary_trt_",Trt1,"_",assay1,".csv",sep="")),
+            row.names = FALSE     
+  )
+}
+box_plot("bindSpike","Vaccine")
+box_plot("bindRBD","Vaccine")
+box_plot("pseudoneutid50","Vaccine")
+box_plot("pseudoneutid80","Vaccine")
+box_plot("bindSpike","Placebo")
+box_plot("bindRBD","Placebo")
+box_plot("pseudoneutid50","Placebo")
+box_plot("pseudoneutid80","Placebo")
+
+
+# baseline Negative placebo arm
+ggplot(dat.long.twophase.sample%>%
+         filter(Trt == "Placebo", 
+                Bserostatus == "Baseline Neg") , aes(y = Day57)) +
+  stat_boxplot(aes(x=assay),
+               geom = "errorbar", 
+               width = 0.5) +  
+  geom_boxplot(aes(x=assay))  +
+  ylim(0,10)
+
+# baseline Negative vaccine arm
+
+ggplot(dat.long.twophase.sample%>%
+         filter(Trt == "Vaccine", 
+                Bserostatus == "Baseline Neg"), aes(y = Day57)) +
+  stat_boxplot(aes(x=assay),
+               geom = "errorbar", 
+               width = 0.5) +  
+  geom_boxplot(aes(x=assay))  +
+  ylim(0,10)
+
+
+
+#Spaghetti plots of Ab markers over time: baseline negative vaccine + placebo arm
+Spaghetti_plot <- function(assay1,Trt1){
+  Spaghettiplot_data <- dat.long.twophase.sample %>%
+    filter(assay == assay1,
+           Bserostatus == "Baseline Neg",
+           Trt == Trt1) 
+  write.csv(data.frame(summary=names(summary(Spaghettiplot_data$Day57)),value=matrix(summary(Spaghettiplot_data$Day57))),
+            here(paste("immuno_graphical/verification/output/output_tester/spaghetti_plots_summary_day57_trt_",Trt1,"_",assay1,".csv",sep="")),
+            row.names = FALSE     
+  )
+  write.csv(data.frame(summary=names(summary(Spaghettiplot_data$Day29)),value=matrix(summary(Spaghettiplot_data$Day29))),
+            here(paste("immuno_graphical/verification/output/output_tester/spaghetti_plots_summary_day29_trt_",Trt1,"_",assay1,".csv",sep="")),
+            row.names = FALSE     
+  )
+  write.csv(data.frame(summary=names(summary(Spaghettiplot_data$Day57)),value=matrix(summary(Spaghettiplot_data$Day57))),
+            here(paste("immuno_graphical/verification/output/output_tester/spaghetti_plots_summary_day57_trt_",Trt1,"_",assay1,".csv",sep="")),
+            row.names = FALSE     
+  )
```

Comment:
is this line necessary? looks like it is in 367-370

Reply:
Deleted.